.github/workflows/build.yml

```diff
@@ -3,147 +3,217 @@ name: CI
 on:
   push:
     branches:
-      - master
-  pull_request:
-    branches:
-      - master
+      - main
+  workflow_dispatch:
+
+# Cancel redundant runs on same ref to save resources
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  # Base release tag pattern; short SHA computed later
+  RELEASE_PREFIX: build
```

Comment:
What is the reason for an env var that has a fixed value and is used exactly once later?